includes/admin/class-wcmp-export.php

```diff
@@ -1031,19 +1031,19 @@ public static function getDigitalStampRangeOptions(): array
     }
 
     /**
-     * @param $chosenMethod
+     * @param  string $chosenMethod
      *
-     * @throws \Exception
+     * @return null|bool|\WC_Shipping_Method
      */
     public static function getShippingMethod(string $chosenMethod)
     {
-        if (version_compare(WOOCOMMERCE_VERSION, "2.6", "<") || $chosenMethod === WCMP_Shipping_Methods::LEGACY_FLAT_RATE) {
+        if ($chosenMethod === WCMP_Shipping_Methods::LEGACY_FLAT_RATE || version_compare(WOOCOMMERCE_VERSION, '2.6', '<')) {
```

Comment:
Wat mij betreft niet blokkerend maar die wooc version check kan er wel uit, we ondersteunen toch niet meer < 5